### PR TITLE
Customize the Smart Contract Address section updates

### PR DESCRIPTION
**Pull request by dragos-rebegea**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [x] other
- update documentation that is currently outdated

### CODE REVIEW

File: docs/developers/tutorials/your-first-dapp.md [hunk 1]
```diff
@@ -236,16 +236,16 @@ cd dapp
...
 
-Let's copy our `config.devnet.tsx` into a new file `config.tsx` and let's fill in the Smart Contract address constant with our smart contract address.
-
 ```sh
-cp src/config.devnet.tsx src/config.tsx
 nano src/config.tsx
 ```
 
-We'll take a look on the line 15:
+We'll take a look on the first instruction
```
Comment (CostinCarabas) [inline docs/developers/tutorials/your-first-dapp.md:243]: Please add `:` at the end of the sentence.

File: docs/developers/tutorials/your-first-dapp.md [hunk 2]
```diff
@@ -236,16 +236,16 @@ cd dapp
...
 
-`export const contractAddress = 'erd1qqqqqqqq....';`
+```sh
+export const contractAddress = 
+  'erd1qqqqqqqqqqqqq...';
+```
 
 and we'll change it to our contract address that was shown after erdpy contract deploy:
```
Comment (CostinCarabas) [inline docs/developers/tutorials/your-first-dapp.md:251]: I would change all the code in the doc file from `this format` to 
```sh
this format
```

**Review by CostinCarabas — CHANGES_REQUESTED** [review]

File: docs/developers/tutorials/your-first-dapp.md [hunk 3]
```diff
@@ -236,16 +236,16 @@ cd dapp
...
 ```sh
-cp src/config.devnet.tsx src/config.tsx
 nano src/config.tsx
 ```
 
-We'll take a look on the line 15:
+We'll take a look on the first instruction
 
-`export const contractAddress = 'erd1qqqqqqqq....';`
+```sh
```
Comment (bogdan-rosianu) [inline docs/developers/tutorials/your-first-dapp.md:245]: this is javascript, bot sh

File: docs/developers/tutorials/your-first-dapp.md (hunk 2 above)
Comment (dragos-rebegea) [reply docs/developers/tutorials/your-first-dapp.md:251]: done

File: docs/developers/tutorials/your-first-dapp.md (hunk 1 above)
Comment (dragos-rebegea) [reply docs/developers/tutorials/your-first-dapp.md]: done

File: docs/developers/tutorials/your-first-dapp.md (hunk 3 above)
Comment (dragos-rebegea) [reply docs/developers/tutorials/your-first-dapp.md]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**